Benchmarking quantized methods

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#29625 Benchmarking quantized methods**

This PR also adds a template for benchmarking methods that require no input.

Differential Revision: [D18443485](https://our.internmc.facebook.com/intern/diff/D18443485)

Commit:
Benchmarking quantized methods

This PR also adds a template for benchmarking methods that require no input.

Commit:
Update on "Benchmarking quantized methods"


This PR also adds a template for benchmarking methods that require no input.

Differential Revision: [D18443485](https://our.internmc.facebook.com/intern/diff/D18443485)

raghuramank100 commented on benchmarks/operator_benchmark/pt/qtensor_method_test.py:
```diff
@@ -0,0 +1,69 @@
+    ],
+    cross_product_configs={
+        'dtype': [torch.quint8],
+        'contig': [True],
```
Please add todo for non-contig
Also, might be a good idea to test this for int32 in addition

raghuramank100 commented on benchmarks/operator_benchmark/pt/qtensor_method_test.py:
```diff
@@ -0,0 +1,69 @@
+qmethods_configs_long = op_bench.cross_product_configs(
+    M=[256, 1024],
+    N=[256, 1024],
+    dtype=[torch.quint8, torch.qint8, torch.qint32],
```
Suggest skipping quint8 and only having qint8 and qint32

Commit:
Update on "Benchmarking quantized methods"


This PR also adds a template for benchmarking methods that require no input.

Differential Revision: [D18443485](https://our.internmc.facebook.com/intern/diff/D18443485)
